components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -197,6 +197,10 @@ export default class PostList extends React.PureComponent {
     }
 
     componentDidUpdate(prevProps, prevState, snapshot) {
+        if (this.props.isMobileView && !prevProps.isMobileView) {
+            this.scrollStopAction = new DelayedAction(this.handleScrollStop);
```

Comment:
Does there need to be an opposite operation here? And if not, what is the risk in having it set unconditionally?

Reply:
No, I don't think there's an opposite of this. I'm not actually sure why this is done conditionally to begin with since a `DelayedAction` doesn't do anything on its own until it's triggered with `fireAfter`.

I think all of the code involving `this.scrollStopAction` could probably have the checks for `isMobileView` removed. The worst thing that would happen then is that we'd be tracking `this.state.isScrolling` on desktop even though it doesn't use it anywhere. That said, I also don't think it makes much difference either way